CoreFoundation/CMakeLists.txt

```diff
@@ -530,10 +533,16 @@ endif()
 
 install(TARGETS
           CoreFoundation
-          CFURLSessionInterface
           CFXMLInterface
         DESTINATION
-          "${CMAKE_INSTALL_FULL_LIBDIR}")
+          "${CMAKE_INSTALL_FULL_LIBDIR}/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
+
+if(FOUNDATION_ENABLE_FOUNDATION_NETWORKING)
+  install(TARGETS
+            CFURLSessionInterface
+          DESTINATION
+            "${CMAKE_INSTALL_FULL_LIBDIR}/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
```

Comment:
I'm concerned that this might negatively impact Windows. Can you please test Windows as well?